.github/workflows/ci-core-lint-reusable.yml

```diff
@@ -19,23 +19,29 @@ jobs:
 
       - name: Start services
         run: |
-          mkdir -p ./volumes/postgres
-          run_retried docker compose pull zk postgres
-          docker compose up -d zk postgres
+          ci_localnet_up
           ci_run sccache --start-server
 
+      - name: Build
+        run: |
+          ci_run bash -c "cd zk_toolbox && cargo build --release"
+          cp ./zk_toolbox/target/release/zk_supervisor ./bin
+          cp ./zk_toolbox/target/release/zk_inception ./bin
+          ci_run bash -c "yarn install"
+          ci_run bash -c "git config --global --add safe.directory /usr/src/zksync"
+
       - name: Setup db
         run: |
-          ci_run zk
-          ci_run run_retried rustup show
-          ci_run zk db migrate
+          ci_run bash -c "mkdir -p /usr/src/zksync/volumes/reth/data"
+          ci_run zk_inception containers --observability false 
+          ci_run zk_supervisor database prepare
```

Comment:
why do you ned it?

Reply:
This is for the `zks database check`: https://github.com/matter-labs/zksync-era/pull/2694/files#diff-3c40f3b56452c5db2daa62cd14db43e11427c1d3392b537d9789458bbeaf2dc8L35-R48